src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,40 @@
+package racingcar.domain;
+
+public class Car {
+    public static final String MOVEMENT_SYMBOL = "-";
+    private static final int INITIAL_POSITION = 0;
+    private static final int CRITERION_FOR_MOVING = 3;
+
+
+    private final CarName name;
+    private int position;
+
+
+    public Car(String name) {
+        this.name = new CarName(name);
+        this.position = INITIAL_POSITION;
+    }
+
+    public void move(int number) {
+        if (number > CRITERION_FOR_MOVING) {
+            position++;
+        }
+    }
+
+    public boolean isWinner(int maxPosition) {
+        return position == maxPosition;
+    }
```

Comment:
메서드 파라미터로 어떤 `position`이 넘어오냐에 따라서 이 메서드 네이밍은 맞을 수도 틀릴 수도 있을 것 같아요. 👀
(예를들어 만약 `minPosition`이 파라미터로 전달된다면 `isLooser`가 되겠죠? )

아래와 같은 네이밍이 조금 더 적절하다고 느껴지는 것 같아요. 😃
```suggestion
    public boolean isSamePosition(int position) {
        return position == position;
    }
```